packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -464,70 +445,66 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
           .pipe(first())
           .subscribe({
             next: (response) => {
-              if (response.status === 201) {
```

Comment:
[openmrsFetch](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-api/src/openmrs-fetch.ts#L35) handles checking the response headers for us, so explicitly checking the status as we're doing here isn't necessary.